.environment/soap_service/soap/port/v2/KvNydr.port

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<soapPort>
+    <id>KvNydr</id>
+    <name>BasicHttpBinding_IUploadFile</name>
+    <projectId>eUAvaN</projectId>
+    <uri>BasicHttpBinding_IUploadFile</uri>
+</soapPort>
```

Comment:
This sets up CastleMock in our dev environment to have a port that mocks PA's SOAP service.